components/Carbon.js

```diff
@@ -377,6 +379,16 @@ function useHighlightLoader() {
   }, [])
 }
 
+let showInvisibleLoaded = false
+function useShowInvisibleLoader() {
```

Comment:
Let's adjust this to use `import()` instead, and only call `import()` if `whiteSpace` is true

Reply:
There is probably no need to check if it is already loaded in this case

Reply:
Sorry @mfix22 i don't know how to do a dynamic import based on boolean value 😢

Reply:
@mfix22 sorry, i don't know how to do dynamic import properly 😞

Reply:
many attempts failed